src/code_80005FD0.c

```diff
@@ -17,6 +17,343 @@
 #include "hud_renderer.h"
 #include "code_80057C60.h"
 
+// Strings, presented by google translate!
+// Note that these are EUC-JP encoded, see:
```

Comment:
Oh no, not more EUC-JP.

Every time I accidentally save over credits.c, I rage quit.

Reply:
Yep, its super annoying. And, unfortunately, this will not be the last file to have it. code_80091750 is going to need it too :(